macsimComponent.cpp

```diff
@@ -60,7 +60,18 @@ macsimComponent::macsimComponent(ComponentId_t id, Params& params)
     m_clock_freq,
     new Clock::Handler<macsimComponent>(this, &macsimComponent::ticReceived));
 
-  m_ptx_core = params.find<bool>("ptx_core", 0);
+  m_acc_core = params.find<bool>("ptx_core", 0);
```

Comment:
delete this line?

Reply:
actually, I'm not sure what I need to do.